test/stdlib/ErrorBridged.swift

```diff
@@ -788,4 +788,23 @@ ErrorBridgingTests.test("error-to-NSObject casts") {
   }
 }
 
+// SR-7732: Casting CFError to Error results in a memory leak
+ErrorBridgingTests.test("CFError-to-Error casts") {
+  func should_not_leak() {
+    let something: Any? = NSError(domain: "Foo", code: 1)
+
+    if let error = something as? Error {
+      expectEqual(
+        "The operation couldn’t be completed. (Foo error 1.)",
```

Comment:
`"The operation couldn\\U2019t be completed. (%@ error %ld.)"` is [localized][] in CoreFoundation. So will this test fail when the user's preferred language isn't English?

[localized]: <https://github.com/apple/swift-corelibs-foundation/blob/653c070d8ee571611ab4cc3e63d640ef4b8d151c/CoreFoundation/Error.subproj/CFError.c#L307>

Reply:
Good catch!  I did actually realize that would be a problem when I wrote it, but forgot to go back and find a better approach.  Thanks for pointing it out!